src/LSST/system/main.cpp

```diff
@@ -0,0 +1,73 @@
+/*
+ *  This file is part of LSST M2 support system package.
+ *
+ * This product includes software developed by the
+ * LSST Project (http://www.lsst.org/).
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the LSST License Statement and
+ * the GNU General Public License along with this program.  If not,
+ * see <http://www.lsstcorp.org/LegalNotices/>.
+ */
+
+#include "system/ComControl.h"
+#include "system/ComControlServer.h"
+#include "system/Config.h"
+#include "util/Log.h"
+
+using namespace std;
+using namespace LSST::m2cellcpp;
+
+int main(int argc, char *argv[]) {
+    LINFO("starting");
+    // TODO: replace "UNIT_TEST" with a real yaml file. DM-34550
+    system::Config::setup("UNIT_TEST");
+
+    // Start the control system
+
+    // Start the telemetry server
+
+    // Start a ComControlServer
+    system::IoContextPtr ioContext = make_shared<boost::asio::io_context>();
+    string strPort = system::Config::get().getValue("server", "port");
```

Comment:
In a certain time point, we would like to use the `yaml` configuration file to hold the `port` information. We would have a command port and a telemetry port.